app/static/js/init.js

```diff
@@ -328,6 +328,29 @@ function addNewCraftingTable() {
   document.getElementById("guesses").appendChild(newTable);
 }
 
+/**
+ * Obtanin the length of the crafting table array
+ * Clear the lats crafting table
+ */
+function clearTable() {
+  let lastTable = craftingTables.length - 1;
+  let table = document.getElementById("tablenumber" + lastTable);
+  let solution = document.getElementById("solutiondiv" + lastTable);
+
+  let solutionSlot = solution.children[0];
+  solutionSlot.removeAttribute("style");
+
+  for (let row = 0; row < craftingTables[lastTable].length; row++) {
+    for (let col = 0; col < craftingTables[lastTable][row].length; col++) {
+      craftingTables[lastTable][row][col] = null;
+
+      let slotId = row * 3 + col;
+      let slot = table.children[slotId].children[0];
+      slot.removeAttribute("style");
```

Comment:
This should use the `setSlotBackground` function for consistency with other functions